contribute/index.md

```diff
@@ -0,0 +1,51 @@
+---
+layout: page
+title: Contribute
+---
+
+{::options parse_block_html="true" /}
+
+So you'd like to support Namecoin?  Great!  There are several ways you can help keep Namecoin strong.
+
+## Donate
+
+Donations help our developers pay the bills, and sometimes can even enable new developers to join the project.  [You can donate to the Namecoin project here.]({{site.baseurl}}donate/)
+
+## Run a full node
+
+Full nodes (Namecoin Core nodes) are the core of the Namecoin P2P network.  The more people run full nodes, and the more geographically and jurisdictionally diverse those full nodes are, the stronger the Namecoin P2P network becomes.  Please consider running [Namecoin Core]({{site.baseurl}}download/#namecoin-core-client-with-qt-name-tab) on as many of the following networks as you can:
+
+* IPv4 (preferably allowing incoming connections)
+* IPv6 (preferably allowing incoming connections)
+* Ephemeral Tor onion service ([docs](https://github.com/namecoin/namecoin-core/blob/master/doc/tor.md))
+* Stable Tor onion service ([docs](https://github.com/namecoin/namecoin-core/blob/master/doc/tor.md))
+
+## Test betas
+
+Do you enjoy living on the cutting edge?  Are you highly skilled at breaking things?  You can help us by testing [beta releases of Namecoin software]({{site.baseurl}}download/betas/), and letting us know what we can improve.
+
+## Run a `.bit` website
+
+Do you have a website?  Register a `.bit` domain for it with Namecoin, enable TLS on it, and tell your website's visitors that they can optionally access your site via Namecoin.
+
+Don't have a website?  Ask your favorite websites' operators to get a `.bit` domain.
+
+## Improve decentralized exchange liquidity
+
+Decentralized exchanges offer better security and privacy than centralized exchanges.  You can help make decentralized exchanges easier to use by posting buy and sell offers on them.  [See the list of decentralized exchanges here.]({{site.baseurl}}exchanges/#decentralized-exchanges)
+
+## Merge-mine Namecoin
+
+Do you mine Bitcoin at a mining pool?  Consider using a Bitcoin mining pool that also merge-mines Namecoin.  You can help keep Namecoin mining decentralized by using the lowest-hashrate pool available that still meets your requirements, or by using [P2Pool](https://github.com/p2pool/p2pool) or solo-mining.
+
+## Write documentation
+
+Is our documentation not clear (or just plain missing) about something?  Are you good at creating text, image, or video tutorials?  You can help us by improving our documentation.  Talk to us (Matrix or IRC preferred) and we'll be happy to assist.
```

Comment:
Maybe move the word about before the parentheses, so it reads: "Is our documentation not clear about (or just plain missing) something?". So that when it is read with the text within the parentheses, it is a little more grammatically correct. But that's just a little nitpick.